nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/connect/client/we/PackageListingProvider.java

```diff
@@ -306,10 +304,10 @@ public Object getDetails(@PathParam("pkgId") String pkgId) {
      * @since 5.6
      * @return true if registration is required for download
      */
-    public boolean registrationRequired(Package pkg) {
-        return pkg.getPackageState() == PackageState.REMOTE && pkg.getType() != PackageType.STUDIO
-                && pkg.getVisibility() != PackageVisibility.PUBLIC && (!ConnectStatusHolder.instance().isRegistred() //
-                || ConnectStatusHolder.instance().getStatus().status() != SubscriptionStatusType.OK);
+    public boolean registrationRequired(DownloadablePackage pkg) {
```

Comment:
Shouldn't we keep the previous method? this is  breaking the backward compatibility.